packages/kbn-ui-shared-deps/entry.js

```diff
@@ -17,31 +17,40 @@
  * under the License.
  */
 
-// import global polyfills before everything else
 require('./polyfills');
 
 // must load before angular
 export const Jquery = require('jquery');
 window.$ = window.jQuery = Jquery;
 
-export const Angular = require('angular');
-export const ElasticCharts = require('@elastic/charts');
-export const ElasticEui = require('@elastic/eui');
-export const ElasticEuiLibServices = require('@elastic/eui/lib/services');
-export const ElasticEuiLightTheme = require('@elastic/eui/dist/eui_theme_light.json');
-export const ElasticEuiDarkTheme = require('@elastic/eui/dist/eui_theme_dark.json');
+// stateful deps
 export const KbnI18n = require('@kbn/i18n');
 export const KbnI18nAngular = require('@kbn/i18n/angular');
 export const KbnI18nReact = require('@kbn/i18n/react');
+export const Angular = require('angular');
 export const Moment = require('moment');
 export const MomentTimezone = require('moment-timezone/moment-timezone');
+export const Monaco = require('./monaco.ts');
+export const MonacoBare = require('monaco-editor/esm/vs/editor/editor.api');
 export const React = require('react');
 export const ReactDom = require('react-dom');
+export const ReactDomServer = require('react-dom/server');
```

Comment:
maybe we could later investigate why is that being used in the client side code 😃